manticore/platforms/evm.py

```diff
@@ -2200,7 +2236,11 @@ def create_account(self, address=None, balance=0, code=None, storage=None, nonce
             raise EthereumError('The account already exists')
 
         if storage is None:
-            storage = self.constraints.new_array(index_bits=256, value_bits=256, name='STORAGE_{:x}'.format(address))
+            # Uninitialized values in a storage are 0 by spec
+            storage = self.constraints.new_array(index_bits=256, value_bits=256, name='STORAGE_{:x}'.format(address), avoid_collisions=True, default=0)
+        else:
+            # TODO: Check storage type/ dict from int/bitvec to int/bitvec
+            pass
```

Comment:
Is this fine to `pass` here? If something is missing, maybe we shall raise an exception (?)